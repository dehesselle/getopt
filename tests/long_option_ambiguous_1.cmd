#!/bin/sh
"$1" -o a -l long1,long2 -- --long
