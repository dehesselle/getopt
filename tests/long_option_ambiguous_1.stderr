$0: option '--long' is ambiguous
